tldw_Server_API/tests/Sharing/test_clone_service.py

```
@@ -0,0 +1,217 @@
+
+import pytest
+
+from tldw_Server_API.app.core.Sharing.clone_service import CloneService
+
+pytestmark = pytest.mark.unit
+
+
+def _make_service(
+    *,
+    src_media_items: dict | None = None,
+    src_transcripts: list | None = None,
```

Ruff (0.15.6)

[warning] Unused function argument: `src_transcripts`

(ARG001)